src/routes/wallet/atomic-swap/tx-proposal.routes.js

```diff
@@ -42,6 +44,12 @@ txProposalRouter.get(
   getLockedUTXOs,
 );
 
+txProposalRouter.post(
```

Comment:
Why is it a POST and not a GET?

Reply:
There will be a refactor based on #279 for this route

Reply:
This route was changed to a `GET` according to design HathorNetwork/rfcs#50 .